src/services/add-markers.js

```diff
@@ -0,0 +1,80 @@
+import { Cartesian3, Color, NearFarScalar } from 'cesium';
+import useStore from '../lib/useStore';
+
+export default function addMarkers(viewer) {
+	// eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Using hooks in arbitrary functions might break the code. Please create a hook to add the markers. Use it like this.


src/lib/hooks/useAddMarkers.js

```
export default function useAddMarkers {

	return function addMarkers(viewer){
		// … code here
	}

}
```

src/components/Map.js

```
function Map() {
	const addMarkers = useAddMarkers();
}
```

Reply:
Done in commit 'Correct Map issues'